python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1435,7 +1435,7 @@ def convert_mean(node, **kwargs):
     mx_axis = node.get("attrs", {}).get("axis", None)
     axes = convert_string_to_list(str(mx_axis)) if mx_axis is not None else None
 
-    keepdims = int(node.get("attrs", {}).get("keepdims", 0))
+    keepdims = int("True" == node.get("attrs", {}).get("keepdims", "False"))
```

Comment:
How about using helper function "get_boolean_attribute_value"?
keepdims = get_boolean_attribute_value(attrs, "keepdims")

Reply:
Take 3 it is